src/App.js

```diff
@@ -1,29 +1,6 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => <Calculator />;
```

Comment:
As required, Let's refactor this to a class-based component ✊

Reply:
> As required, Let's refactor this to a class-based component fist_raised

Hello @bimbolabuari 
Thank you for your review
I have tried with class-based component for App
But it was giving me linter errors, That it should be pure function and i had to change it again to function-based component
Can you please help me with that ?